src/test/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloudIT.java

```diff
@@ -391,20 +450,25 @@ private static InstanceTemplate createTemplate(Map<String, String> googleLabels)
         return instanceTemplate;
     }
 
+    private static InstanceConfiguration snapshotInstanceConfiguration() {
```

Comment:
Suggest adding a case to test snapshot creation for windows VMs as well.

Reply:
Done.